packages/babel-types/src/constants.js

```diff
@@ -38,6 +38,7 @@ export const NUMBER_BINARY_OPERATORS = [
 ];
 export const BINARY_OPERATORS = [
   "+",
+  "??",
```

Comment:
This belongs in `LOGICAL_OPERATORS`, not `BINARY_OPERATORS`

Reply:
Ah, forgot to move it. Thanks.